Description: Upstream doesn't use gcc. The gcc in Trusty- doesn't have stdatomic.h
  See https://bugs.launchpad.net/oxide/+bug/1668614

Index: dev/third_party/ffmpeg/libavutil/cpu.c
===================================================================
--- dev.orig/third_party/ffmpeg/libavutil/cpu.c
+++ dev/third_party/ffmpeg/libavutil/cpu.c
@@ -18,7 +18,12 @@
 
 #include <stddef.h>
 #include <stdint.h>
+// Ubuntu 14.04 gcc doesn't provide stdatomic.h, so use the compat version
+#if defined(__GNUC__) && __GNUC__ == 4 && __GNUC_MINOR__ == 8
+#include <compat/atomics/gcc/stdatomic.h>
+#else
 #include <stdatomic.h>
+#endif
 
 #include "attributes.h"
 #include "cpu.h"
